lib/rules/require-parentheses-around-arrow-param.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Requires parentheses around arrow function expressions with a single parameter.
```

Comment:
Maybe just `ES6: Requires parentheses...` or we add another category like `Version: ES6` similar to `Type, Version`